internal/snmp/translate_test.go

```diff
@@ -87,3 +89,54 @@ func TestTrapLookupFail(t *testing.T) {
 		})
 	}
 }
+
+type TestingMibLoader struct {
+	folders []string
+	files   []string
+}
+
+func (t *TestingMibLoader) appendPath(path string) {
+	t.folders = append(t.folders, path)
+}
+
+func (t *TestingMibLoader) loadModule(path string) error {
+	t.files = append(t.files, path)
+	return nil
+}
+func TestFolderLookup(t *testing.T) {
+	paths := [][]string{
+		{"testdata"},
+		{"testdata", "loadMibsFromPath"},
+		{"testdata", "loadMibsFromPath", "linkTarget"},
+		{"testdata", "loadMibsFromPath", "root"},
+		{"testdata", "mibs"},
+	}
+	tests := []struct {
+		name    string
+		folders []string
+		files   []string
+	}{
+		{
+			name:  "loading folders",
+			files: []string{"emptyFile", "testmib"},
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			loader := TestingMibLoader{}
+
+			err := LoadMibsFromPath([]string{"testdata"}, testutil.Logger{}, &loader)
+			require.NoError(t, err)
+			for _, pathSlice := range paths {
+				path := filepath.Join(pathSlice...)
+				tt.folders = append(tt.folders, path)
+			}
+			require.Equal(t, tt.folders, loader.folders)
+			if runtime.GOOS == "windows" {
+				tt.files = []string{"emptyFile", "symlink", "testmib"}
+			}
```

Comment:
This special case for windows doesn't make sense if this is going to be a table test. It could be moved into the table or we could have a separate test for windows since symbolic links don't work the same way on windows.

Reply:
I changed the format to match what I found in another test (config_test.go). I am not certain this is the correct way to do it, if not let me know and I can change it :)